src/components/main/StaffList.js

```diff
@@ -4,12 +4,56 @@ import './StaffList.css';
 
 class StaffList extends React.Component {
    
-  
+    listAdd = (index,sf) =>{
+        let quantity = parseInt(this.props.staffs[index].quantity)
+        let shiftTotal = parseInt(this.props.staffs[index].shiftTotal)
+        let shift = parseInt(this.props.staffs[index].shift)
+        quantity = quantity + 1
+
+        if (shift == 12) {
+            shiftTotal = parseInt(shiftTotal) + 12;
+        }else{
+            shiftTotal = parseInt(shiftTotal) + 8;
+        }
+       
+       if(quantity <= 100){
+           this.props.staffs[index].quantity = quantity
+           this.props.staffs[index].shiftTotal = shiftTotal
+            this.props.parent.getChildrenMsg(this.props.staffs)
+       }
+       
+    }
+
+    listSub = (index, sf) =>{
+        let quantity = parseInt(this.props.staffs[index].quantity)
+        let shiftTotal = parseInt(this.props.staffs[index].shiftTotal)
+        let shift = parseInt(this.props.staffs[index].shift)
+        quantity = quantity - 1;
+
+        if (shift == 12) {
+            shiftTotal = parseInt(shiftTotal) - 12;
+        }
+        else {
+            shiftTotal = parseInt(shiftTotal) - 8;
+        }
+
+       if(quantity <= 0){
+           this.props.staffs.splice(index,1)
+       }else{
+           this.props.staffs[index].quantity = quantity
+           this.props.staffs[index].shiftTotal = shiftTotal
+       }
+       this.props.parent.getChildrenMsg(this.props.staffs)
+    }
 
     render() {
 
         const staffList = this.props.staffs.map((staff, i) =>
             <Row key={staff.id} id={staff.id} className="border">
+                <Col className="border">
+                    <text className="btext" onClick={this.listAdd.bind(staff,i)}>+</text> 
+                    <text className="btext2" onClick={this.listSub.bind(staff,i)}>-</text>
```

Comment:
when you add a staff member, an error comes up on the Developer Tools console that the tag 'text' isn't recognized. While you're developing, you should be opening Developer Tools and looking at the console to check for warnings and errors.

Reply:
Also, I don't think these should be text tags anyways. They aren't places where you are inputting anything. They should be spans or icons or something. If they weren't text tags, then this would also help resolve the cursor I'm seeing when I hover over the fields. This should be a pointer cursor, which you will still have to add to CSS, as I mentioned in my comment on your CSS file.